tests/testnet.spec.js

```diff
@@ -11,12 +11,12 @@ const fetchJson = async (uri, opts = {}) => {
 /**
  * Please set your private/public keys and existing fioAddresses
  */
-let privateKey = '',
-  publicKey = '',
-  privateKey2 = '',
-  publicKey2 = '',
-  testFioAddressName = '',
-  testFioAddressName2 = ''
+ let privateKey = '',
+   publicKey = '',
+   privateKey2 = '',
+   publicKey2 = '',
+   testFioAddressName = '',
+   testFioAddressName2 = ''
```

Comment:
please remove these spaces, these lines shouldn't change

Reply:
ill fix it